### ISSUE TITLE
Mac shows character suggestions when holding down certain keys

### ISSUE THREAD

**Issue by BigZaphod** [openfl/openfl#1146]
I'm not sure which component is to blame for this - for all I know, it might be SDL's fault - but it seems there must be an offscreen text field that is used to capture the input and it's not disabling all of the various autocomplete/suggestion modes. Screenshot of what happens when holding down "a" (also happens for many other characters that have alternate forms:

<img width="441" alt="screen shot 2016-05-24 at 8 27 20 pm" src="https://cloud.githubusercontent.com/assets/453721/15525291/6ce91bdc-21ee-11e6-9265-e58dd2686f68.png">

Running OS X 10.11.5, Lime 2.9.1, OpenFL 3.6.1, Flixel 4.0.1

**Comment by jgranick** [comment]
Weird! This is firmly in SDL's camp, I see fixes related to caps lock on Mac, and some text input changes since the version we use currently, but nothing that immediately jumps out to me as a fix. I'll try and see if they've seen this before, thank you!

**Comment by BigZaphod** [comment]
Just to add some more confusion to the mix, I noticed after messing around with our other experiments in the other issue thread that suddenly I wasn't seeing these popups anymore on Mac. I reverted everything back to the release versions to try to track it down:

```
$ haxelib list
actuate: [1.8.6]
box2d: [1.2.3]
flixel-addons: [2.0.0]
flixel-demos: [2.1.0]
flixel-templates: [2.0.1]
flixel-tools: [1.1.3]
flixel-ui: [2.0.0]
flixel: [4.0.1]
haxelib_client: [3.2.0-rc.3]
hxcpp: [3.3.47]
lime-samples: [2.6.0]
lime: [2.9.1]
openfl-samples: [3.3.1]
openfl: [3.6.1]
```

And then the popups came back. I then tried compiling my project with -Dnext and... the popups are gone with *no* other changes. Literally the only change is adding -Dnext. Removing -Dnext and the popups come back. So.. yeah. :)
